awesomplete.js

```diff
@@ -304,6 +304,8 @@ function Suggestion(data) {
 	  ? { label: data[0], value: data[1] }
 	  : typeof data === "object" && "label" in data && "value" in data ? data : { label: data, value: data };
```

Comment:
- [x] As @vlazar mentioned, you don't need this unless data is an object. Since there is another check about whether this is the case, we should convert it to a proper block if and do it in there.

```suggestion
	if (Array.isArray(data)) {
		o = { label: data[0], value: data[1] }
	} else if (typeof data === "object") {
		for (var property in data) {
			this[property] = data[property];
		}
	} else if (typeof data === "object" && !("label" in data && "value" in data)) {
		o = { label: data, value: data }
	}
```